src/transformers/models/persimmon/modeling_persimmon.py

```diff
@@ -72,7 +72,7 @@ def _expand_mask(mask: torch.Tensor, dtype: torch.dtype, tgt_len: Optional[int]
 
 
 # Copied from transformers.models.llama.modeling_llama.LlamaRotaryEmbedding with Llama->Persimmon
-class PersimmonRotaryEmbedding(torch.nn.Module):
+class PersimmonRotaryEmbedding(nn.Module):
```

Comment:
same here

Reply:
This is the code after running `make fix-copies`, this change because transformers in the past have some bug, so this is the automatic code to pass the test on github, we should keep it.

Reply:
Sounds good, thanks @pphuc25